server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/ServiceDeploymentServiceElasticsearch.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.server.elasticsearch;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.hawkular.apm.api.model.analytics.BuildStamp;
+import org.hawkular.apm.api.model.analytics.ServiceDeployment;
+import org.hawkular.apm.api.services.ServiceDeploymentService;
+
+/**
+ * The implementation of {@link ServiceDeploymentService} for the Elasticsearch backend.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public class ServiceDeploymentServiceElasticsearch implements ServiceDeploymentService {
+
+    /**
+     * @see ServiceDeploymentService#getServiceDeployments()
+     */
+    @Override
+    public List<ServiceDeployment> getServiceDeployments() {
+        // TODO: do the actual implementation :)
+        List<ServiceDeployment> serviceDeployments = new ArrayList<>(3);
+        serviceDeployments.add(new ServiceDeployment("foo", Collections.singletonList(new BuildStamp("foo-1"))));
+        serviceDeployments.add(new ServiceDeployment("bar", Collections.singletonList(new BuildStamp("bar-1"))));
+        serviceDeployments.add(new ServiceDeployment("baz", Collections.singletonList(new BuildStamp("baz-1"))));
+        return serviceDeployments;
+    }
+
+    @Override
+    public ServiceDeployment getService(String serviceName) {
```

Comment:
For consistency, should probably take the Criteria as well, as the list of buildstamps for the service may be dependent upon the filter criteria.